src/Providers/LumenServiceProvider.php

```diff
@@ -93,9 +98,14 @@ protected function registerAuthProvider()
      */
     protected function registerStorageProvider()
     {
-        $this->app->singleton('tymon.jwt.provider.storage', function ($app) {
-            return $this->getConfigInstance('providers.storage');
-        });
+        $this->app->singleton(
+            'tymon.jwt.provider.storage',
+            function ($app) {
+                $provider = $this->config('providers.storage');
+
+                return $app->make($provider, [new CacheManager($app)]);
```

Comment:
This avoid generating a new instance instead of resolving with what already bind to the IoC.